src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_RMSMappingQuality.java

```diff
@@ -51,6 +51,18 @@
     private static final OneShotLogger allele_logger = new OneShotLogger(AS_RMSMappingQuality.class);
     private static final OneShotLogger genotype_logger = new OneShotLogger(AS_RMSMappingQuality.class);
 
+    /**
+     * Get all the raw key names for this annotation
+     *
+     * @return a non-empty list
+     */
+    @Override
+    public List<String> getRawKeyNames() {
+        final List<String> allRawKeys = new ArrayList<>(Arrays.asList(getPrimaryRawKey()));
+        allRawKeys.addAll(getSecondaryRawKeys());
+        return allRawKeys;
```

Comment:
Might be a good opportunity to push this code up into the interface as a default implementation -- that way you can be sure that all implemetations of the interface will by default be calling `getPrimaryRawKey()`/`getSecondaryRawKeys()` directly and in the right order, and most/all annotation classes won't even need to implement their own version of `getRawKeyNames()`

Reply:
To the interface?  It told me that interface methods had to be abstract.

Reply:
@ldgauthier Here's an example showing how to include a default implementation of a method in an interface: https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html

Reply:
Eg., 

```
public interface TimeClient {
    default ZonedDateTime getZonedDateTime(String zoneString) {
        return ZonedDateTime.of(getLocalDateTime(), getZoneId(zoneString));
    }
```